### PR TITLE
Enhancement: Add userdata parameter to automatically select the pen tool.

### PR DESCRIPTION
**Pull request by Scroody**
### What does this PR do?

This PR adds a custom parameter that allow the viewer to automatically enter the multi user whiteboard with the draw tool selected using: 'userdata-pen_select=true'.

### Closes Issue(s)

#19604

### CODE REVIEW

File: bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ const currentParameters = [
   'bbb_mirror_own_webcam',
   // PRESENTATION
   'bbb_force_restore_presentation_on_new_events',
+  'pen_select',
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js:54]: Please prepend it with `bbb_`.

File: docs/docs/administration/customize.md [hunk 2]
```diff
@@ -1496,7 +1496,8 @@ Useful tools for development:
 | ----------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------------- |
 | `userdata-bbb_multi_user_pen_only=` | If set to `true`, only the pen tool will be available to non-participants when multi-user whiteboard is enabled | `false`       |
 | `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`                                                         | all enabled   |
-| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`                                      | all enabled   |
+| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`
```
Comment (antobinary) [inline docs/docs/administration/customize.md:1499]:
```suggestion
| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`                                      | all enabled   |
```

File: docs/docs/administration/customize.md [hunk 3]
```diff
@@ -1497,6 +1497,7 @@ Useful tools for development:
 | `userdata-bbb_multi_user_pen_only=` | If set to `true`, only the pen tool will be available to non-participants when multi-user whiteboard is enabled | `false`       |
 | `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`                                                         | all enabled   |
 | `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`                                      | all enabled   |
+| `userdata-bbb_pen_select=`    | If set to true, allows viewers to start multi-user whiteboard with pen tool selected.                                      | false  |
```
Comment (antobinary) [inline docs/docs/administration/customize.md:1500]:
```suggestion
| `userdata-bbb_pen_select=`    | If set to true, the defult tool for viewers when multi-user whiteboard is activated will be the pen tool.                                      | false  |
```

**Comment by antobinary** [comment]
We're discussing the overall approach. Possibly #19604 should be solved at UI level on breakouts create, not via params passing.

**Comment by Scroody** [comment]
Might not be useful anymore

**Pull request closed without merging**